index.html

```diff
@@ -1 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+</head>
+<body>
+    <Main id="main">
+        <div id ="title">
+            <h1>Dr. Norman Borlaug</h1>
+        </div>
+        <figure id="img-div">
+            <img id ="image"></img>
+        </figure>
+        <div id = "img-caption">
+            <p>He was a great Doctor</p>
+        </div>
+        <div id="tribute-info">
+            <p>HE was an awesome man</p>
+        </div>
+    </Main>
 
```

Comment:
avoid leaving whitespaces in your code

Reply:
Kindly phrase all comments politely, as suggestions instead of orders so that the author will not feel coarse. For example:

* This is a polite suggestion: *"In order to make the code base a lot more readable, kindly avoid leaving unnecessary whitespaces"*
* This reads as an order: *"Avoid leaving whitespaces in your code"*